terra-jupyter-python/tests/smoke_test.ipynb

```diff
@@ -85,9 +85,9 @@
    "metadata": {},
    "outputs": [],
    "source": [
-    "output = !pip3 show pendulum\n",
+    "output = !pip3 show emoji\n",
```

Comment:
This test checks whether we can install a new package - in the new image, `pendulum` is already installed, so it makes the test fail. Switch to a package that is not already installed.